pysat/_instrument.py

```diff
@@ -1053,77 +1051,83 @@ def _assign_funcs(self, by_name=False, inst_module=None):
             # Either flags are not specified, or this combo is not
             self._password_req = False
 
-    def __str__(self):
+    def __repr__(self):
+        # Print the basic Instrument properties
+        out_str = "".join(["Instrument(platform='", self.platform, "', name='",
+                           self.name, "', sat_id='", self.sat_id,
```

Comment:
```suggestion
        out_str = "".join(["pysat.Instrument(platform='", self.platform, 
                           "', name='", self.name, "', sat_id='", self.sat_id,
```

Reply:
I don't agree with this suggestion.  Following the example of numpy, once you have declared a class object such as an array, the __repr__ looks like: `array('string', dtype='<U6')` and no reference to the path to get to the class is made.

Reply:
You are right about the numpy standard. I checked a few others and results are mixed:
```
In [14]: np.array([0, 1, 2]).__repr__()                                                                                                                               
Out[14]: 'array([0, 1, 2])'

In [8]: dt.datetime(2009, 1, 1).__repr__()                                                                                                                            
Out[8]: 'datetime.datetime(2009, 1, 1, 0, 0)'

In [11]: xarray.Dataset().__repr__()                                                                                                                                  
Out[11]: '<xarray.Dataset>\nDimensions:  ()\nData variables:\n    *empty*'

In [13]: pds.DataFrame().__repr__()                                                                                                                                   
Out[13]: 'Empty DataFrame\nColumns: []\nIndex: []'

```
I do still prefer including the pysat but I am ok with leaving it off.